assets/js/modules/pagespeed-insights/setup.js

```diff
@@ -88,7 +88,7 @@ class PageSpeedInsightsSetup extends Component {
 
 	handleAPIKeyChange( e ) {
 		this.setState( {
-			apikey: e.currentTarget.value,
+			apikey: e.currentTarget.value.replace( /\s/g, '' ),
```

Comment:
Quick follow-up @aaemnnosttv, why this and not `.trim()`?

Reply:
This prevents any whitespace from being entered anywhere in the key. `trim` only removes from either end. `trim` would also work as a solution for the reported issue, but I figured the key  would never have any whitespace, and that is what the brief describes as well (stripping any whitespace).

If you'd prefer to go with `trim` that's ok with me; it's a quick change.

Reply:
I'd prioritize consistency with https://github.com/google/site-kit-wp/blob/develop/assets/js/components/setup-wizard/wizard-step-client-credentials.js#L127. I understand your reasoning for using `replace`, but I'd argue then it should also happen that way in the linked line.
The argument for `trim` would be that it's easier to understand and that problems with spaces realistically only occur in the beginning or end anyway, like you said.

Reply:
Sounds good @felixarntz - I didn't realize there was a precedent here. I've updated the line to use `trim` now instead 👍